resources/frontend.rb

```diff
@@ -19,8 +19,19 @@
       variables['frontend'][new_resource.name] ||= {}
       variables['frontend'][new_resource.name]['default_backend'] ||= ''
       variables['frontend'][new_resource.name]['default_backend'] << new_resource.default_backend
-      variables['frontend'][new_resource.name]['bind'] ||= ''
-      variables['frontend'][new_resource.name]['bind'] << new_resource.bind
+      if new_resource.bind.is_a? Hash
+        variables['frontend'][new_resource.name]['bind'] = []
```

Comment:
I know the ||= but I'm worried about typing the variable.
For proper rendering, I would prefer to be sure of an Array.
Right suggestion for duplication, I'll fix immediately :)